pyomo/contrib/fbbt/interval.py

```diff
@@ -140,7 +168,7 @@ def power(xl, xu, yl, yu):
             msg += 'The upper bound of a variable raised to the power of {0} is {1}'.format(yl, xu)
             raise InfeasibleConstraintException(msg)
         xl = 0
-        lb, ub = power(xl, xu, yl, yu)
+        lb, ub = power(xl, xu, yl, yu, feasibility_tol)
```

Comment:
Why is this clause (including `yl == yu` above) not relaxed by `feasibility_tol`?

Reply:
The purpose of this clause is to catch constant exponents only (e.g., `m.x**1.5`). We do not want to catch things like `m.x**m.y` where `1 <= m.y <= 1+1e-4`. In that case, the `m.y` is not guaranteed to be fractional, so we cannot assert that m.x has to be positive. Relaxing `yl == yu` with `feasibility_tol` is actually not a relaxation.